ga4gh/datamodel/__init__.py

```diff
@@ -12,8 +12,11 @@
 import tempfile
 import shutil
 import atexit
+import collections
+
 
 import ga4gh.exceptions as exceptions
+import ga4gh.serverconfig as serverconfig
```

Comment:
We don't want to import this here, as this only gives us static configuration --- see below for a suggestion on how to restructure to give us dynamic config.